adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java

```diff
@@ -90,26 +90,26 @@ protected void setUp() throws Exception {
         // all of them
         // until it finds the correct one for ADAL broker.
         for (Signature signature : info.signatures) {
-            testSignature = signature.toByteArray();
+            mTestSignature = signature.toByteArray();
             MessageDigest md = MessageDigest.getInstance("SHA");
-            md.update(testSignature);
-            testTag = Base64.encodeToString(md.digest(), Base64.NO_WRAP);
+            md.update(mTestSignature);
+            mTestTag = Base64.encodeToString(md.digest(), Base64.NO_WRAP);
             break;
```

Comment:
since you are updating this, you can remove the for loop as we break at the end.